You need to fix a test:
`FAILED tests/components/mqtt/test_state_vacuum.py::test_default_supported_features[hass_config0] - AssertionError: assert ['battery', '...tart', 'stop'] == ['battery', '...atus', 'stop']`

--- a/tests/components/mqtt/test_state_vacuum.py
+++ b/tests/components/mqtt/test_state_vacuum.py
@@ -112,7 +112,7 @@ async def test_default_supported_features(
     entity = hass.states.get("vacuum.mqtttest")
     entity_features = entity.attributes.get(mqttvacuum.CONF_SUPPORTED_FEATURES, 0)
     assert sorted(services_to_strings(entity_features, SERVICE_TO_STRING)) == sorted(
-        ["start", "stop", "return_home", "battery", "status", "clean_spot"]
+        ["start", "stop", "return_home", "battery", "clean_spot"]
     )
 
 

